Show URL of currently selected bookmark #16

LukasJanik55 opened the pull request:
Hello, sending the PR.

Resolves #16

exhuma commented on src/components/sh-link.ts:
```diff
@@ -101,13 +101,24 @@ class Link extends LitElement {
+    let title = html`
+      <a
+        tabindex="1"
+        class="underline text-blue-900 dark:text-blue-100 testA"
```
Looks like the `testA` here can be removed.

exhuma commented on src/components/sh-link.ts:
```diff
@@ -101,13 +101,24 @@ class Link extends LitElement {
     };
   }
 
+  onMouseEvent(text: string) {
```
I would rename this to something else because it is not _only_ happening on mouse events. Something like `dispatchFocusChange` or `dispatchTitleChange`.

exhuma reviewed (commented):
The changes look good. Just a couple of microscopic changes, and in addition to the small comments about the code, I would retarget this to merge into `develop` instead of `master`. It seems I have forgotten to set `develop` as default branch target. I will do this immediately :)

exhuma commented:
You know what, the changes are super small, I'll quickly fix those myself :smile:

exhuma commented:
I pushed the new branch [issue-16-integration](https://github.com/exhuma/schmackhaft/tree/issue-16-integration) with a minor change. The change ensures that the "status bar" also shows up properly when using the code as a browser extension.

There's one issue though: The "focus" event is not firing when using the keyboard for navigation. To reproduce, focus the quicksearch text field and use the arrow-up/down keys.

Could you have a look at that please?

As a pointer, look at this piece which deals with the keyboard navigation: https://github.com/exhuma/schmackhaft/blob/0a9dccd992640f7bcfd124dfcdcd23821d732b13/src/components/sh-bookmarklist.ts#L51

exhuma commented:
I forgot to mention: The new branch is rebased onto `develop`. So if you could rebase your branch onto that one it would provide a cleaner merge later on.

LukasJanik55 commented:
I am not sure if i did everything right, but i've:

1. changed the PR target branch to development
2. pulled the changes
3. added functionality for keyboard controls
4. pushed everything into my branch

It should be working now correctly under all conditions (mouse hover, tab focus and keyboard control). If you find anything wrong or a need of change, let me know.

PS: sorry again for not responding sooner, i'll be here daily if you need any more changes
